sync-data-to-another-database/README.md

```diff
@@ -0,0 +1,195 @@
+# Synchronize Kuzzle with another database
+
+## Introduction
+
+Kuzzle uses Elasticsearch as a database, which allows it to offer very good search performance on large volumes.  
+
+However, it can also be useful to dump Kuzzle data into more traditional databases, for example if you wanted to store and analyse historical data.  
+[Cassandra](https://cassandra.apache.org/) is a distributed NoSQL database designed to handle large volumes of data.  
+
+In this How-To, we will show you how to develop a Kuzzle Plugin that synchonizes Kuzzle's Elasticsearch database with Cassandra.  
+For this example we will use data from the NYC Yellow Taxi dataset.  
+
+## Architecture
+
+We will be using the Kuzzle stack (Kuzzle, Elasticsearch and Redis) along with an additional container that runs the Cassandra database. (Check [docker-compose.yml](docker-compose.yml) for more details)
+
+On Kuzzle, the data will be stored in the `yellow-taxi` collection of the `nyc-open-data` index according to the following mapping:
+
+```js
+{
+  "pickup_datetime":  { "type": "date", "format": "MM/dd/yyyy hh:mm:ss a" },
+  "dropoff_datetime": { "type": "date", "format": "MM/dd/yyyy hh:mm:ss a" },
+  "passenger_count":  { "type": "long" },
+  "trip_distance":    { "type": "double" },
+  "pickup_position":  { "type": "geo_point" },
+  "dropoff_position": { "type": "geo_point" },
+  "fare_amount":      { "type": "double" }
+}
+```
+
+On Cassandra's side, we will dump the data into the `yellow_taxi` table of the `nyc_open_data` keyspace. (Note the use of `_` instead of `-` because of Cassandra's restrictions)  
+
+Elasticsearch has defined a specific type to store a coordinate, in order to emulate this type in Cassandra we will also create a [User Defined Type](https://docs.datastax.com/en/cql/3.3/cql/cql_using/useCreateUDT.html) named `geopoint` which corresponds to the type in Elasticsearch.  
```

Comment:
Proposal (to be validated with @appacea): 

In Elasticseach we use the `geo_point` type to index our documents geographically. With Cassandra, we will have to create a  [User Defined Type](https://docs.datastax.com/en/cql/3.3/cql/cql_using/useCreateUDT.html) emulating that type, and we will name it `geopoint`.

Reply:
I don't see how the `geopoint` user type is created in Cassandra, can you add it?

Reply:
It's a little lower, in the CQL requests.

Reply:
I agree with @scottinet proposed changes:

"In Elasticseach we use the geo_point type to index our documents geographically. With Cassandra, we will have to create a User Defined Type emulating that type, and we will name it geopoint."

Reply:
Fixed.